src/sentry/web/frontend/react_page.py

```diff
@@ -107,12 +106,20 @@ def handle_auth_required(self, request: Request, *args, **kwargs) -> HttpRespons
         # For normal users, let parent class handle (e.g. redirect to login page)
         return super().handle_auth_required(request, *args, **kwargs)
 
-    def handle(self, request: Request, organization, **kwargs) -> HttpResponse:
-        if "project_id" in kwargs and request.GET.get("onboarding"):
-            project = Project.objects.filter(
-                organization=organization, slug=kwargs["project_id"]
-            ).first()
-            first_event_pending.send(project=project, user=request.user, sender=self)
+    def handle(self, request: Request, organization: RpcOrganization, **kwargs) -> HttpResponse:
+        if "project_slug" in kwargs and "onboarding" in request.GET:
```

Comment:
Is `onboarding` ever a falsey value? Previously the logic would not pass for falsey values of `onboarding` but now it will.